README.md

```diff
@@ -45,4 +45,16 @@ On the picture is presented an example of simulator work with the rule ``Templat
 If the income message is wrong (not ``NewOrderSingle``), the rule in simulator will not generate an outgoing message. 
 If the income message is correct (``NewOrderSingle``), the rule will generate one ``ExecutionReport``.
 
-![picture](scheme.png)
\ No newline at end of file
+![picture](scheme.png)
+
+## Release notes
+
+### 3.2.0
+
++ Update th2-sim to version 3.7.0
+  + Added `IRuleContext.removeRule()` method which allows a rule to remove itself
+  + Added ability to schedule execution of arbitrary actions via `IRuleContext.execute` methods
++ Update th2-common to version 3.19.0
+    + Extended Utility classes
+        + Added the toTreeTable method to convert message/message filter to event data
+        + Added the Event.exception method to include an exception and optionally all the causes to the body data as a series of messages
```

Comment:
```suggestion
+ Update th2-common to version 3.19.0
    + Update `th2-grpc-common` and `th2-grpc-service-generator` versions to `3.2.0` and `3.1.12` respectively
    + Disable waiting for connection recovery when closing the `SubscribeMonitor`
    + Change the way channels are stored (they mapped to the pin instead of the thread).
  It might increase the average number of channels used by the box, but it also limits the max number of channels to the number of pins
    + Added the property `workers`, which changes the count of gRPC server's threads
    + Added `session alias` and `direction` labels to incoming metrics
    + Rework logging for incoming and outgoing messages
    + Resets embedded `log4j` configuration before configuring it from a file
    + Fixed a bug with message filtering by `message_type`
```